lib/base.js

```diff
@@ -18,16 +18,20 @@
 */
 
 module.exports = {
-    extends: 'semistandard',
+    extends: 'standard',
 
     env: {
-        // `semistandard` sets this to true. We reset it here,
+        // `standard` sets this to true. We reset it here,
         // to make the config extend logic clearer.
         node: false
     },
 
     rules: {
         indent: ['error', 4],
-        camelcase: 'off'
+        camelcase: 'off',
+
+        // semistandard
```

Comment:
```suggestion
        // We replicate the eslint-config-semistandard rules here instead of
        // extending that config, since it is poorly maintained.
```

Reply:
+1